src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala

```diff
@@ -93,7 +93,12 @@ class SetMultiTargets()(implicit messageBusPublisher: MessageBusPublisher) exten
 
     db.run(dbAct.transactionally).flatMap { scheduled =>
       Future.traverse(scheduled){ device =>
-        messageBusPublisher.publish(UpdateSpec(namespace, device, UpdateStatus.Pending))
+        messageBusPublisher.publish(DeviceUpdateEvent(
```

Comment:
Would it make sense to use `publishSafe` here? Or we can move on if we fail to publish the message?

Reply:
I just realised there's many `publish` like this. I guess they're not critical.